src/app/portal/components/hearing/hearing.component.ts

```diff
@@ -109,6 +122,7 @@ export class HearingComponent implements OnInit {
   case$ = this.caseService.getCase(this.caseId).pipe(shareReplay(1));
   hearing$ = this.caseService.getHearingById(this.caseId, this.hearingId);
   audio$ = this.hearingService.getAudio(this.hearingId);
+  annotations$ = this.hearingService.getAnnotations(this.hearingId);
```

Comment:
I think you need to prevent the observable call here if the user is not a judge or admin to prevent a 403 returning from the backend to error out the application

Reply:
Now fixed